CONTRIBUTING.md

```diff
@@ -1,128 +1,82 @@
-# Contributing to QUIC
+# QUIC version 1 is done
 
-Anyone can contribute to QUIC; you don't have to join the Working Group, because there is no "membership" -- anyone who participates in the work, as outlined below, is part of the QUIC Working Group.
+The base-drafts repository is the historical home of the QUIC version 1
+specifications that were written by the QUIC Working Group.
 
-Before doing so, it's a good idea to familiarize yourself with our current [issues list](https://github.com/quicwg/base-drafts/issues) and [charter](https://datatracker.ietf.org/wg/quic/about/). If you're
-new to this, you may also want to read the [Tao of the IETF](https://www.ietf.org/tao.html).
+The set of documents are:
 
-**Be aware that all contributions fall under the "NOTE WELL" terms outlined below.**
+* [QUIC: A UDP-Based Multiplexed and Secure Transport](https://tools.ietf.org/html/draft-ietf-quic-transport)
+* [Version-Independent Properties of QUIC](https://tools.ietf.org/html/draft-ietf-quic-invariants)
+* [Using TLS to Secure QUIC](https://tools.ietf.org/html/draft-ietf-quic-tls)
+* [QUIC Loss Detection and Congestion Control](https://tools.ietf.org/html/draft-ietf-quic-recovery)
+* [Hypertext Transfer Protocol Version 3 (HTTP/3)](https://tools.ietf.org/html/draft-ietf-quic-http)
+* [QPACK: Header Compression for HTTP/3](https://tools.ietf.org/html/draft-ietf-quic-qpack)
```

Comment:
The URLs should eventually be changed to the archival URLs at rfc-editor.org.

Reply:
I think that we should create our own renderings of the final RFCs and point to those.  The official renderings are terrible.